prompts-dr.md

```
@@ -0,0 +1,230 @@
+4. [Prompt — Job: Build](#4-prompt--job-build)
+5. [Prompt — Job: Deploy](#5-prompt--job-deploy)
+6. [Prompt — Seguridad y Secrets](#6-prompt--seguridad-y-secrets)
+7. [Prompt — Consolidación Final](#7-prompt--consolidación-final)
+
+---
+
+## 1. Contexto y Rol
+
+**Rol asignado al modelo:**
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)